product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.4-Modify-payload-by-removing-elements/src/test/java/org.wso2.carbon.ei.scenario.test/RemoveElementsTest.java

```diff
@@ -102,8 +101,44 @@ public void removeContentOfElementUsingEnrichMediator() throws IOException, XMLS
                         + "   </soapenv:Body>"
                         + "</soapenv:Envelope>";
 
-        HTTPUtils.invokeSoapActionAndAssert(url, request, ScenarioConstants.MESSAGE_ID, expectedResponse, 200,
-                "urn:mediate", "RemoveContentOfElementUsingEnrichMediator");
+        HTTPUtils.invokeSoapActionAndAssert(url, request, testCaseId, expectedResponse, 200,
+                "urn:mediate", "removeContentOfElementUsingEnrichMediator");
+    }
+
+    //This test is to verify if payload can be modified by removing content of an element using xslt mediator.
```

Comment:
Shall we add a doc comment instead of a single line comment?

Reply:
This is done.